docs/SPECIFICATION.md

```diff
@@ -424,6 +427,49 @@ A LinkRemove in a message `m` is valid only if it passes these validations:
 
 1.  `m.data.type` must be `MESSAGE_TYPE_LINK_REMOVE`
 
+## 1.7 Username Proof
```

Comment:
`USER_DATA_TYPE_FNAME` has been renamed to `USER_DATA_TYPE_USERNAME` in the UserDataAdd spec above.